### DISCUSSION TITLE
parameter format

### DISCUSSION THREAD

**Discussion by IceBreakerSours** [snl-dakota/dakota#105] [Q&A] [answered]
Hello, 

I am trying to have Dakota write out parameters in a specific format. For instance, Fw.d in Fortran 77 - where F is a real number, w and d are integer constants that denote field width and number of significant digits, respectively. I tried searching the forums and the documentation but haven't run into anything helpful so far. Do you have any pointers for me? 

I am using dprepro and Dakota 6.10 repo revision c963f6c.

**Comment by IceBreakerSours** [comment, marked as answer]
dprepro --help addressed my question.

**Comment by briadam** [comment]
Glad you resolved your issue. In addition to the global format option available at the command line, you can also format each substitution separately if you like. For this case, I believe The docs for dakota.interfacing dprepro module should apply: https://snl-dakota.github.io/docs/6.19.0/users/usingdakota/inputfile/interfaces/dakotainterfacing.html#per-field-output-formatting
